demos-europe/demosplan-core
# make addons activateable

This PR is fixing all the little bugs in Core to enable activating addons. It only means, that the project *can* load with an activated addon, not that it will necessarily work.

The addon will not yet do anything as there is more to be done with the FE integration, but it should be fully present after these changes.

### How to review/test
On this branch, it should be possible to install an addon **and** activate it while the project is still working fine. 

### Linked PRs (optional)
- Can only be merged after [this addon PR](https://github.com/demos-europe/demosplan-addon/pull/19)
